pkg/cmd/pipelinerun/describe.go

```diff
@@ -84,6 +84,10 @@ or
 					if err != nil {
 						return err
 					}
+					if len(prs) == 0 {
+						fmt.Fprintf(s.Out, fmt.Sprintf("No PipelineRuns present in namespace %s\n", opts.Params.Namespace()))
```

Comment:
The build test is failing due to your use of `fmt.Sprintf`. Just use the following:

```go
fmt.Fprintf(s.Out, "No PipelineRuns present in namespace %s", opts.Params.Namespace())
```

Reply:
done